recipes-devtools/gcc-for-nvcc/gcc-8-crosssdk.inc

```diff
@@ -8,5 +8,5 @@ SYSTEMLIBS1 = "${SDKPATHNATIVE}${libdir_nativesdk}/"
 
 GCCMULTILIB = "--disable-multilib"
 
-DEPENDS = "virtual/${TARGET_PREFIX}binutils-crosssdk gettext-native ${NATIVEDEPS}"
-PROVIDES = "virtual/${TARGET_PREFIX}cuda-gcc-crosssdk virtual/${TARGET_PREFIX}cuda-g++-crosssdk"
+DEPENDS = "virtual/${TARGET_PREFIX}binutils gettext-native ${NATIVEDEPS}"
+PROVIDES = "virtual/${TARGET_PREFIX}cuda-gcc-crosssdk virtual/${TARGET_PREFIX}cuda-g++"
```

Comment:
Shouldn't the `-crosssdk` suffix be dropped for both of the entries in this PROVIDES?

Reply:
Right, forgot this one as in https://git.yoctoproject.org/poky/diff/meta/recipes-devtools/gcc/gcc-crosssdk.inc?id=768c89e306d0486acf8bc76cecef90436f6551b3
Sorry I need to manually change the patch and when I see it building don't look any more.
I will update and fix.